app/assets/javascripts/discourse/tests/acceptance/composer-messages-test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  acceptance,
+  exists,
+  query,
+} from "discourse/tests/helpers/qunit-helpers";
+import { click, triggerKeyEvent, visit } from "@ember/test-helpers";
+import { test } from "qunit";
+
+acceptance("Composer - Messages", function (needs) {
+  needs.user();
+  needs.pretender((server, helper) => {
+    server.get("/composer_messages/user_not_seen", () => {
+      return helper.response({
+        user_count: 1,
+        warning_message:
+          "The person you are messaging, charlie, hasn’t been seen here in a very long time – about 1 year ago. They may not receive your message. You may wish to seek out alternate methods of contacting charlie.",
```

Comment:
Can we use `I18n.t()` here? It would be shorter and match changes to localization in the long run.

Reply:
I'm not sure if we can use server-side locales on client-side tests. I tried and there were errors. Also the main thing we need to verify here is that the error message received from the server response is shown as-is on client side.

Reply:
Ah, right, I didn't realize the description was localized server-side. Maybe we don't need the message to be localized server-side? It would probably be a tiny bit more efficient to return just the list of usernames that haven't been seen in a while and then let the client do the message handling.

Just an option, not a blocker.

Reply:
Sure, I just went ahead and localized the message on the client side.